biome.json

```diff
@@ -5,17 +5,27 @@
 		"indentWidth": 2,
 		"lineWidth": 100,
 		"formatWithErrors": true,
-		"ignore": ["*.astro"]
-	},
-	"organizeImports": {
-		"enabled": true
+		"includes": ["**", "!**/*.astro"]
```

Comment:
What’s the reason to disable `organizeImports` for `.astro` files? It looks like this PR _is_ organizing them. Or do I misunderstand this glob?